pyiron_atomistics/atomistics/structure/neighbors.py

```diff
@@ -1047,3 +1047,120 @@ def get_cluster(dist_vec, ind_vec, prec=prec):
+            positions (numpy.ndarray): The cartesian positions of the trajectories
+            cells (numpy.ndarray/None): The varying cell shapes
+            num_neighbors (int): The cutoff for the number of neighbors
+            **kwargs (dict): Additional arguments to be passed to the `get_neighbors()` routine
+                             (eg. cutoff_radius, norm_order , etc.)
+        """
+        super().__init__()
+        self._init_structure = init_structure
+        self._neighbor_indices = None
+        self._neighbor_distances = None
+        self._neighbor_vectors = None
+        self._positions = positions
+        self._cells = cells
+        self._num_neighbors = num_neighbors
+        self._get_neighbors_kwargs = kwargs
+
+    @property
+    def neighbor_indices(self):
+        """
+        Neighbour indices (excluding itself) of each atom computed using the get_neighbors_traj() method
+
+        Returns:
+
+            numpy.ndarray/list: An array of dimension N_steps / stride x N_atoms x N_neighbors
+
+        """
+        return self._neighbor_indices
+
+    @property
+    def neighbor_distances(self):
+        """
+        Neighbour distances (excluding itself) of each atom computed using the get_neighbors_traj() method
+
+        Returns:
+
+            numpy.ndarray/list: An array of dimension N_steps / stride x N_atoms x N_neighbors
+
+        """
+        return self._neighbor_distances
+
+    @property
+    def neighbor_vectors(self):
+        """
+        Neighbour vectors (excluding itself) of each atom computed using the get_neighbors_traj() method
+
+        Returns:
+
+            numpy.ndarray/list: An array of dimension N_steps / stride x N_atoms x N_neighbors x 3
+
+        """
+        return self._neighbor_vectors
+
+    @property
+    def num_neighbors(self):
+        """
+        The maximum number of neighbors to be computed
+
+        Returns:
+
+            int: The max number of neighbors
+        """
+        return self._num_neighbors
+
+    def compute_neighbors(self):
+        """
+        Compute the neighbors across the trajectory
+        """
+        self._neighbor_indices, self._neighbor_distances, self._neighbor_vectors = \
+            _get_neighbors(self._init_structure,
+                           positions=self._positions,
+                           cells=self._cells,
+                           num_neighbors=self._num_neighbors, **self._get_neighbors_kwargs)
+
+    def to_hdf(self, hdf, group_name="neighbors_traj"):
+        """
+        Store this `NeighborTraj` instance in a .h5 file.
+
+        Args:
+            hdf (pyiron_base.generic.hdfio.FileHDFio): HDF5 group or file
+            group_name (str): Name of the HDF5 group
+
+        """
+        super().to_hdf(hdf=hdf, group_name=group_name)
+
+
+def _get_neighbors(struct, positions, cells=None, num_neighbors=20, **kwargs):
```

Comment:
```suggestion
def _get_neighbors(struct, positions, cells=None, num_neighbors=12, **kwargs):
```
Consistent defaults make it a bit easier to read.  I have no opinions on 12 vs. 20, but we should pick one.